Magento_Sales/templates/order/items.phtml

```diff
@@ -93,39 +93,39 @@
 
                                 <div
                                     class="order-gift-message"
-                                    id="order-item-gift-message-<?= $giftId ?>"
+                                    id="order-item-gift-message-<?= $block->escapeHtmlAttr($giftId) ?>"
                                     role="region"
                                     aria-expanded="false"
                                     tabindex="-1"
                                 >
                                     <a
                                         href="#"
                                         title="<?= $block->escapeHtmlAttr(__('Close')) ?>"
-                                        aria-controls="order-item-gift-message-<?= $giftId ?>"
-                                        data-item-id="<?= $giftId ?>"
+                                        aria-controls="order-item-gift-message-<?= $block->escapeHtmlAttr($giftId) ?>"
+                                        data-item-id="<?= $block->escapeHtmlAttr($giftId) ?>"
                                         class="link">
-                                        <?= __('Close') ?>
+                                        <?= $block->escapeHtml(__('Close')) ?>
                                     </a>
 
                                     <dl class="item-options">
                                         <dt class="item-sender">
                                             <strong class="label">
-                                                <?= __('From') ?>
+                                                <?= $block->escapeHtml(__('From')) ?>
                                             </strong>
 
                                             <?= $block->escapeHtml($giftMessage->getSender()) ?>
                                         </dt>
 
                                         <dt class="item-recipient">
                                             <strong class="label">
-                                                <?= __('To') ?>
+                                                <?= $block->escapeHtml(__('To')) ?>
                                             </strong>
 
                                             <?= $block->escapeHtml($giftMessage->getRecipient()) ?>
                                         </dt>
 
                                         <dd class="item-message">
-                                            <?= $escapeGiftMessage ?>
+                                            <?= $block->escapeHtml($escapeGiftMessage) ?>
```

Comment:
We should update it to
`<?= /* @noEscape */ $this->helper(\Magento\GiftMessage\Helper\Message::class)->getEscapedGiftMessage($item) ?>` as in original file